src/contracts/util.ts

```diff
@@ -4,72 +4,29 @@ export const networkPassphrase =
   import.meta.env.PUBLIC_NETWORK_PASSPHRASE ??
   "Standalone Network ; February 2017";
 
-import { contractMapping, XAssetSymbol } from "./contractConfig";
 import { Client, Errors } from "xBTC";
 export type XAssetContract = Client;
 export const ContractErrors = Errors;
 
-const contractClientMap = {
-  xBTC: new Client({
-    networkPassphrase,
-    contractId: contractMapping.xBTC,
-    rpcUrl,
-    publicKey: undefined,
-  }),
-  xETH: new Client({
-    networkPassphrase,
-    contractId: contractMapping.xETH,
-    rpcUrl,
-    publicKey: undefined,
-  }),
-  xUSDT: new Client({
-    networkPassphrase,
-    contractId: contractMapping.xUSDT,
-    rpcUrl,
-    publicKey: undefined,
-  }),
-  xXRP: new Client({
-    networkPassphrase,
-    contractId: contractMapping.xXRP,
-    rpcUrl,
-    publicKey: undefined,
-  }),
-  xSOL: new Client({
-    networkPassphrase,
-    contractId: contractMapping.xSOL,
-    rpcUrl,
-    publicKey: undefined,
-  }),
-  xADA: new Client({
-    networkPassphrase,
-    contractId: contractMapping.xADA,
-    rpcUrl,
-    publicKey: undefined,
-  }),
-  xDOT: new Client({
+// Get contract client instance by symbol
+export const getContractBySymbol = (symbol: string, contractMapping: Record<string, string>): XAssetContract => {
+  const contractAddress = contractMapping[symbol];
+  if (!contractAddress) {
+    throw new Error(`Contract not found for symbol: ${symbol}`);
+  }
+
+  return new Client({
```

Comment:
Right now, this leaves a lone `xBTC` in environments.toml in order to create a client to use here.